tests/integration/05.Dfw.Tests.ps1

```diff
@@ -615,6 +618,17 @@ Describe "DFW" {
 
         it "Can create an l3 rule with an vnic based source" {
             # Update PowerNSX to get nics fix to arrays
+            $vm1vnic = Get-Vm $testVMName1 | Get-NetworkAdapter 
+            $rule = $l3sec | New-NsxFirewallRule -Name "pester_dfw_rule1" -source $vm1vnic -action allow
+            $rule.sources.source.type | should be "Vnic"
+            ($rule.sources.source.name).count | should be 2
```

Comment:
we shouldnt be testing multiple nics in this test - count should be one and the test needs to be restructured to reflect